### DISCUSSION TITLE
Invalid python interpreter but only on Github actions?

### DISCUSSION THREAD

**Discussion by ChristopherRabotin** [PyO3/maturin#1677] [Q&A] [answered]
In this example, https://github.com/nyx-space/nyx/actions/runs/5352420082/jobs/9707520777?pr=187, the docker image fails to build the project. This is a sudden change: it didn't happen five days ago.

Python is installed and the command to start the docker image is:

`CMD ["maturin", "build", "-F", "python", "--release", "-i", "python3"]`

I will also say that it does not seem to happen locally. 

Any idea how I could debug this?

Thanks

**Comment by messense** [comment]
Make sure you have `python3` executable in `PATH`.

**Comment by ChristopherRabotin** [comment, marked as answer]
The fact that it suddenly didn't work is a hint I should have understood immediately since I'm using opensuse tumbleweed. It turns out that python 3.10 is no longer in the path as `python3` but as `python3.10`.

Using `-i python3.10` was sufficient to fix the issue.

As such, the command in the docker container becomes:
`CMD ["maturin", "build", "-F", "python", "--release", "-i", "python3.10"]`
